src/lightning_app/__version__.py

```diff
@@ -1 +1 @@
-version = "0.6.0rc0"
```

Comment:
Isn't https://github.com/Lightning-AI/lightning/releases/tag/app%2F0.6.0rc0 already released?

Reply:
no, we have pushed just RC